packages/kit/src/runtime/server/utils.js

```diff
@@ -6,7 +6,7 @@ import { negotiate } from '../../utils/http.js';
 import { fix_stack_trace } from '../shared-server.js';
 import { ENDPOINT_METHODS } from '../../constants.js';
 import { escape_html } from '../../utils/escape.js';
-import { with_event } from '../app/server/event.js';
+import { with_event } from '../../exports/internal/event.js';
```

Comment:
importing from `../exports` will likely result in `with_event` being bundled multiple times

```suggestion
import { with_event } from '@sveltejs/kit/internal';
```